sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/GoneAndRetryWithRetryPolicy.java

```diff
@@ -226,12 +226,12 @@ private CosmosException logAndWrapExceptionWithLastRetryWithException(Exception
         }
 
         private Pair<Mono<ShouldRetryResult>, Boolean> handleGoneException(GoneException exception) {
-            logger.info("Received gone exception, will retry, {}", exception.toString());
+            logger.debug("Received gone exception, will retry, {}", exception.toString());
```

Comment:
I don't think the info ones are the problem.

mostly I have seen two cases which confuses the user and creates CRIs for us
1. the last retry attempt in GoneAndRetryPolicy which is warning. Even though GoneAndRetryPolicy gets exhausted ClientRetryPolicy may retry
2. I have seen container create not exists shows an error message 409 if the container exists from the retry policy

Reply:
We want warning on last retry from GoneAndRetryPolicy , even if ClientRetryPolicy retry again , because at that time we might be going in different region so warning is fair log here